### DISCUSSION TITLE
The algorithm of grain boundary curvature

### DISCUSSION THREAD

**Discussion by Brian-Liu-rx** [mtex-toolbox/mtex#949] [Ask Anything] [answered]
Hi，

    I am a beginner working on mtex and I wish to know this great toolbox very well.
    I am curious about how the curvature of grain boundary is calculated by mtex. Is there any paper or documentation about the algorithm of it?
    Regards，

**Comment by kilir** [comment, marked as answer]
Hi,
it is detailed here: https://www.github.com/mtex-toolbox/mtex/tree/develop/EBSDAnalysis%2F%40grainBoundary%2Fcurvature.m
Hope that helps.
Cheers, 
Rüdiger

**Reply by Brian-Liu-rx** [reply]
Thanks，Rüdiger

And another one is how can I set the color of "notindexed" cell be black or other color. The defult is white.

![image](https://user-images.githubusercontent.com/69452222/114655437-c97e2800-9d1e-11eb-8945-b3ae73f7baca.png)

**Reply by ralfHielscher** [reply]
Have a look here https://mtex-toolbox.github.io/EBSDPlotting.html

**Reply by ralfHielscher** [reply]
It would be very helpful for us if you could give us some suggestions how we can change the documentation such that those things are more easy to find.

Ralf.
